app/res/values/strings.xml

```diff
@@ -306,6 +306,7 @@
     <string name="keep_changes" cc:translatable="true">SAVE INCOMPLETE</string>
     <string name="go_to_location" cc:translatable="true">Go to Location</string>
     <string name="font_size" cc:translatable="true">Text Font Size</string>
+    <string name="button_font_size" cc:translatable="true">Button Font Size</string>
     <string name="change_font_size" cc:translatable="true">Text Font Size</string>
     <string name="form_scan_starting" cc:translatable="true">Scanning for forms...</string>
     <string name="form_scan_finished" cc:translatable="true">Form scanning completed</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/e245f76be76f1b4dd06e3e59d5ee621f84f2e5b0/app/res/values/strings.xml#L312"></span>The resource <code>R.string.form_scan_finished</code> appears to be unused</td>
    </tr>
  </tbody>
</table>